### ISSUE TITLE
Using transpose with chorus recall=true transposes recalled choruses twice

### ISSUE THREAD

**Issue by lpinner** [sciurius/chordpro#41]
Using transpose with chorus recall=true transposes the recalled chorus a second time.  For example, in the test below the chords in the recalled chorus are being transposed -2 (+10) instead of +5.  If `{transpose: +2}` is used, the recalled chorus would be transposed +4.
```
$ chordpro --version
This is ChordPro version 0.94
$ chordpro -X -o test.pdf --config test.json test.pro

```

test.pro (see also attached [test.zip](https://github.com/sciurius/chordpro/files/1734157/test.zip) with test.pro/json/pdf)
```
{transpose: +5}
{title: Swing Low Sweet Chariot}

{Chorus}
{start_of_chorus}
Swing [D]low, sweet [G]chari[D]ot,
Comin’ for to carry me [A7]home.
Swing [D7]low, sweet [G]chari[D]ot,
Comin’ for to [A7]carry me [D]home.
{end_of_chorus}

I [D]looked over Jordan, and [G]what did I [D]see,
Comin’ for to carry me [A7]home.
A [D]band of angels [G]comin’ after [D]me,
Comin’ for to [A7]carry me [D]home.

{Chorus}

```
test.json
```
{
    "pdf" : {
        "chorus" : {
            "tag" : "Chorus",
            "recall" : {
                "tag"   : "Chorus",
                "type"  : "comment",
                "quote" : true,
            },
        },
    },
}
```
test.pdf

![image](https://user-images.githubusercontent.com/4827816/36345672-9c3c8e80-1483-11e8-8f48-3a77f3462128.png)

**Comment by sciurius** [comment]
Good catch. Fixed in dev (0.940_005).
Thanks for the excellent, concise and complete bug report.

**Comment by lpinner** [comment]
Thanks for the speedy fix. Tested and working fine in dev on a couple of my songs.
